app/templates/components/article-featured-video.hbs

```diff
@@ -1,17 +1,28 @@
-<div class="video-container{{unless isPlayerLoading ' player-ready'}}{{if isPlayed ' is-played'}}"
-	 style="background-image: linear-gradient(to top, rgba(0, 0, 0, 0.9) 0%, transparent 100%), url({{model.url}});">
-	{{wds-spinner active=isPlayerLoading overlay=false radius=27 strokeWidth=5}}
-	<div {{action 'playVideo'}} class="video-placeholder">
-		<div class="video-details">
-			<div class="video-label">
-				Watch
-				<span class="video-time">{{videoTime}}</span>
+<div class="video-container">
+	{{! The 'video-wrapper' element is needed here to fix an issue with background color while
+	scrolling on iOS 9. }}
+	<div class="video-wrapper">
+		{{wds-spinner active=isPlayerLoading overlay=false radius=27 strokeWidth=5}}
+		<img src="{{model.url}}" class="video-thumbnail" {{action 'playVideo'}}>
+		<div {{action 'playVideo'}} class="video-placeholder">
+			<div class="video-details">
+				<div class="video-label">
+					Watch
+					<span class="video-time">{{videoTime}}</span>
+				</div>
+				<div class="video-title">{{videoTitle}}</div>
+			</div>
+			<div class="video-play-circle">
+				{{#if isVideoDrawerVisible}}
+					{{svg 'wds-icons-play-triangle' class='wds-icon wds-icon-tiny'}}
```

Comment:
other way to do it would be
```hbs
{{svg 'name' class=(concat 'wds-icon wds-icon-' (if isVideoDrawerVisible 'tiny' 'small'))}}
```

not sure if that's more readable though, I remember that blocked if was 'slow', not sure if that is still a problem